src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+    render = () => (
+      <div className="grid container max-width-sm">
+        <p value={0} className="result">0</p>
```

Comment:
- [OPTIONAL] Following the [given design](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A), you could improve the display number on the calculator, why not make a color white text, better center, and bigger? That'd improve your project.
![image](https://user-images.githubusercontent.com/20563252/130460825-35ba8c67-c30c-48be-9ba3-29fd085c9082.png)
![image](https://user-images.githubusercontent.com/20563252/130460995-8eb00852-b2c0-4fe4-a6f7-7c291cb23ff6.png)